doc/specs/stdlib_stats_distribution_exponential.md

```diff
@@ -31,6 +36,10 @@ The algorithm used for generating exponential random variates is fundamentally l
 
 `result = ` [[stdlib_stats_distribution_exponential(module):rvs_exp(interface)]] `([lambda] [[, array_size]])`
```

Comment:
I suggest to remove this one from the specs or to mention that it is deprecated.